pan3d/ui/file_select.py

```diff
@@ -0,0 +1,102 @@
+from trame.widgets import html, vuetify3 as vuetify
+from trame.app import get_server
+
+server = get_server()
+
+
+class FileSelect(vuetify.VCard):
+    def __init__(
+        self,
+        import_function,
+        export_function,
+        dialog_shown="dialog_shown",
+        selected_config_file="selected_config_file",
+        state_export="state_export",
+        dialog_message="dialog_message",
+    ):
+        def submit_import():
+            files = server.state[selected_config_file]
+            if files and len(files) > 0:
+                file_content = server.state[selected_config_file][0]["content"]
+                import_function(file_content)
+
+        super().__init__()
+        with self:
+            with vuetify.VCardText(v_show=dialog_shown):
+                vuetify.VBtn(
+                    flat=True,
+                    icon="mdi-close",
+                    style="float: right",
+                    click="%s = undefined" % dialog_shown,
+                )
+                vuetify.VCardTitle(
+                    "{{ %s }}" % dialog_message,
+                    v_show=dialog_message,
+                )
+                with html.Div(v_show="!%s" % dialog_message):
```

Comment:
f-string

Reply:
Changed in [28561cb](https://github.com/Kitware/pan3d/pull/33/commits/28561cb331878bed9f5131df5a01d8307360c41e)